src/repr/src/explain_new/mod.rs

```diff
@@ -666,6 +666,36 @@ where
     Ok(())
 }
 
+/// Pretty-prints a list of indices.
+#[derive(Debug)]
+pub struct Indices<'a>(pub &'a [usize]);
```

Comment:
I think the old copy of this in (the remaining) `explain.rs` could be removed. `explain.rs` could just import the new copy.

Reply:
I wanted to avoid this work as very briefly I will wipe `explain.rs` altogether.